tests need python modules that are not installed yet: point the built ones.
Reported upstream: https://bugreports.qt.io/browse/PYSIDE-1430

--- a/sources/pyside2/tests/init_paths.py	2020-11-11 13:51:30.000000000 +0100
+++ b/sources/pyside2/tests/init_paths.py	2020-11-21 23:36:33.131282473 +0100
@@ -75,12 +75,12 @@
 
     python_dirs = [os.path.join(src_dir, 'util')]  # Helper module
 
-    pyside_build_dir = os.path.join(get_build_dir(), 'pyside2')
-    python_dirs.append(pyside_build_dir)   # for PySide2
+    pyside_build_dir = os.path.join(get_build_dir(), 'PySide2')
+    python_dirs.append(get_build_dir())   # for PySide2
     lib_dirs = [os.path.join(pyside_build_dir, 'libpyside')]
 
     if testbindings_module:
-        python_dirs.append(os.path.join(pyside_build_dir,
+        python_dirs.append(os.path.join(get_build_dir(),
                                         'tests', 'pysidetest'))
         lib_dirs.append(_get_qt_lib_dir())
 
